components/library/src/content/ser.rs

```diff
@@ -178,6 +180,7 @@ impl<'a> SerializingPage<'a> {
         SerializingPage {
             relative_path: &page.file.relative,
             ancestors,
+            plain_content: &page.plain_content,
```

Comment:
That approach is a no-go, it will make Zola roughly twice slower since it has to serialize the content twice in the HTML case.

Reply:
Alright, I'll have to either find a way to only serialize one of them or to reuse `page.content`. Didn't realize serializing is the bottleneck.